diff.py

```diff
@@ -1080,13 +1090,14 @@ def search_map_file(
         if len(cands) == 1:
             return cands[0]
     elif project.map_format == "mw":
+        section_patten = config.diff_section or r"\.(?:init|text)"
```

Comment:
nit: section_pattern

I wonder what the .init thing here is about... seems like it doesn't work given our .text restriction. I'd be inclined to remove it and see if anyone yells

use `re.escape(config.diff_section)` to avoid the `.` being interpreted as a regex part